python_http_client/client.py

```diff
@@ -171,10 +181,15 @@ def _make_request(self, opener, request, timeout=None):
         """
         timeout = timeout or self.timeout
         try:
-            return opener.open(request, timeout=timeout)
+            return (opener.open(request, timeout=timeout),
+                    request.get_method())
         except HTTPError as err:
             exc = handle_error(err)
             exc.__cause__ = None
+            _logger.info('{method} Response: {status} {body}'.format(
```

Comment:
Should make this a `debug` log and move to above `handle_error`, in case that fails.